pom.xml

```diff
@@ -272,6 +272,7 @@
 				<artifactId>maven-surefire-plugin</artifactId>
 				<version>${maven-surefire-plugin.version}</version>
 				<configuration>
+					<argLine>-Dfile.encoding=UTF-8</argLine>
```

Comment:
I think we should add same for maven-failsafe-plugin as well, either <encoding>UTF-8</encoding> or <argLine>-Dfile.encoding=UTF-8</argLine>